test/runtests.jl

```diff
@@ -9,9 +9,8 @@ include("utils.jl")
     include_test("differentiation.jl")
 
     @testset "Ambiguities" begin
-        # TODO: reduce the number of ambiguities for Julia 1.6
         if VERSION.prerelease == () && !Sys.iswindows() && VERSION < v"1.6.0"
-            @test length(Test.detect_ambiguities(ManifoldsBase)) <= 65
+            @test length(Test.detect_ambiguities(ManifoldsBase)) <= 66
```

Comment:
BTW, does this still need to be bumped?

Reply:
I think it would be good since it introduces a small new feature, yes.

Reply:
OK.